src/Storages/MergeTree/IMergeTreeDataPart.h

```diff
@@ -218,7 +218,7 @@ class IMergeTreeDataPart : public std::enable_shared_from_this<IMergeTreeDataPar
     /// Compute part block id for zero level part. Otherwise throws an exception.
     /// If token is not empty, block id is calculated based on it instead of block data
     UInt128 getPartBlockIDHash() const;
-    String getNewPartBlockID(std::string_view token) const;
```

Comment:
The only remaining use case for this is to write logs. Do we really want to leave this?